sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -271,7 +271,7 @@ case class AlterTableUnsetPropertiesCommand(
       }
     }
     // If comment is in the table property, we reset it to None
-    val tableComment = if (propKeys.contains("comment")) None else table.properties.get("comment")
+    val tableComment = if (propKeys.contains("comment")) None else table.comment
```

Comment:
Oh. I see. It goes to override the original table metadata if no comment is set.